README.md

```diff
@@ -0,0 +1,2984 @@
+Note that some spoken dialog is missing from the game texts.
+
+**Gabriel Knight CD:** Speech and text can be toggled via the "Text" and
+"Voice" buttons in the game's settings dialog, or via ScummVM's audio
+options.
+
+**King's Quest 6 CD:** Speech and text can be toggled via the "Mode"
+button in the options dialog (with an extra "Dual" setting added in
+ScummVM), or via ScummVM's audio options.
+
+**King's Quest VII CD:** There is no in-game option to toggle speech and
+text. Only ScummVM's audio options can be used to toggle this feature.
+Note that the subtitles were disabled in the official release of this
+game, so some subtitles may be wrong or missing.
+
+**Laura Bow 2 CD:** Speech and text can be toggled via the "Mode" button
+in the options dialog (with an extra "Dual" setting added in ScummVM),
+or via ScummVM's audio options.
+
+**Leisure Suit Larry 6 CD:** Either speech only or speech and text can
+be selected. There is no in-game option to toggle text only. Only
+ScummVM's audio options can be used to enable the text only mode.
+
+**Leisure Suit Larry 6 (hires) CD:** Text can be toggled by selecting
+the "Text On/Off" option from the in-game "Game" menu, or via ScummVM's
+audio options. Speech cannot be disabled.
+
+**Police Quest 4 CD:** Either speech only or text only can be selected
+from the game's settings dialog. Only ScummVM's audio options can be
+used to enable text+speech mode.
+
+**Shivers CD:** Text can be toggled by selecting the "Text" option from
+the game's settings dialog, or via ScummVM's audio options. Note that
+only videos have subtitles in this game.
+
+**Space Quest 4 CD:** Speech and text can be toggled via the "Display
+Mode" button in the options dialog, or via ScummVM's audio options.
+
+**Space Quest 6 CD:** Speech and text can be toggled via the "Speech"
+and "Text" buttons in the game's settings dialog, or via ScummVM's audio
+options.
+
+**Torin's Passage CD:** Text can be toggled by selecting "Closed
+Captioning" from the in-game "Game" menu. Speech can be disabled by
+selecting "Audio Mixer" from the in-game "Game" menu and setting the
+speech volume to zero.
+
+### 3.29) Zork games notes
+
+To run the supported Zork games (Zork Nemesis: The Forbidden Lands and
+Zork: Grand Inquisitor) you need to copy some (extra) data to its
+corresponding destination.
+
+#### 3.29.1) Zork Nemesis: The Forbidden Lands
+
+Download the Liberation(tm) fonts package
+<https://releases.pagure.org/liberation-fonts/liberation-fonts-ttf-2.00.1.tar.gz>
+and unpack all the ttf files into your ScummVM extras directory.
+Alternatively, download the GNU FreeFont TTF package
+<https://ftp.gnu.org/gnu/freefont/freefont-ttf.zip> and unzip all the
+ttf files from the sfd directory into your ScummVM extras directory,
+though at the time of writing these fonts cause some text rendering
+issues. Download the subtitles patch
+<https://www.thezorklibrary.com/installguides/znpatch.zip> and unzip the
+addon directory
```

Comment:
NT4 does not have the same heading level as the other Windows versions.

(Very obsolete versions of Windows could probably be removed -- but not in the PR)